config/config.go

```diff
@@ -1509,6 +1509,17 @@ var (
 	// and label name characters that are not alphanumerics/underscores to underscores.
 	// Unit and type suffixes may be appended to metric names, according to certain rules.
 	UnderscoreEscapingWithSuffixes translationStrategyOption = "UnderscoreEscapingWithSuffixes"
+	// NoTranslation (EXPERIMENTAL): disables all translation of incoming metric
+	// and label names.
+	//
+	// Note that because metrics in Open Telemetry are considered
```

Comment:
[Nit] For consistency:
```suggestion
	// Note that because metrics in OpenTelemetry are considered
```